Gruntfile.js

```diff
@@ -466,14 +465,9 @@ module.exports = function(grunt) {
       'start-webdriver': {
         cmd:
           'mkdir -p tests/logs && ' +
-          './node_modules/.bin/webdriver-manager update && ' +
-          './node_modules/.bin/webdriver-manager start > tests/logs/webdriver.log & ' +
+          'docker run -d -p 4444:4444 -v /dev/shm:/dev/shm selenium/standalone-chrome:3.141.59-yttrium > tests/logs/webdriver.log & ' +
```

Comment:
Looks like the [docs ](https://github.com/SeleniumHQ/docker-selenium#--shm-size2g) removed the volume mounting in favor of setting a cli arg. 

`docker run -d -p 4444:4444 --shm-size="2g" selenium/standalone-chrome:3.141.59-yttrium > tests/logs/webdriver.log &`